Makefile.include

```diff
@@ -709,10 +709,13 @@ distclean:
 	-@for i in $(USEPKG) ; do "$(MAKE)" -C $(RIOTPKG)/$$i distclean ; done
 	-@rm -rf $(BINDIRBASE)
 
+# Include PROGRAMMER_FLASH/PROGRAMMER_RESET variables
+include $(RIOTMAKE)/tools/programmer.inc.mk
```

Comment:
Looks good, only comment remaining on my side: is `PROGRAMMER` the right name? (can't think of another myself)